server/src/api/cards.ts

```diff
@@ -240,3 +240,64 @@ export function ranks(url: string) {
 		},
 	]
 }
+
```

Comment:
Please put this in `hof.ts` or something because theres gonna be similar methods!